mypy-strict.ini

```diff
@@ -40,6 +40,8 @@ files =
     tools/autograd/*.py,
     tools/clang_tidy.py,
     tools/codegen/*.py,
+    tools/codegen/api/*.py,
+    tools/codegen/dest/*.py,
```

Comment:
Actually, I now see that you could just do this instead, which is even simpler:
```suggestion
    tools/codegen,
```

Reply:
ah yep, way better thanks!